.vscode/settings.json

```diff
@@ -0,0 +1,34 @@
+{
+  "cSpell.language": "en-US",
+  "editor.rulers": [
+    80
+  ],
+  "git.rebaseWhenSync": true,
+  "python.formatting.provider": "black",
+  "python.jediEnabled": true,
+  "python.linting.flake8Enabled": true,
+  "python.linting.mypyArgs": [
+    "--config-file=tox.ini",
+    "--follow-imports=skip"
+  ],
+  "python.linting.mypyEnabled": true,
+  "python.linting.pydocstyleArgs": [
+    "--match='(?!test_).*\\.py'"
+  ],
+  "python.linting.pydocstyleEnabled": true,
+  "python.linting.pylintCategorySeverity.refactor": "Information",
+  "python.linting.pylintEnabled": true,
+  "python.linting.pylintUseMinimalCheckers": false,
+  "python.testing.pytestArgs": [ // https://github.com/microsoft/vscode-python/issues/693
+    "tests",
+    "--no-cov",
+    // "--cov-config=tests/.coveragerc",
```

Comment:
I would remove those comment lines, and add it once you want code coverage to be in there